### ISSUE TITLE
parameters.csv

### ISSUE THREAD

**Issue by HedvigS** [glottobank/pygrambank#4]
It'd be great if parameters.csv also include a column for "possible values", in this fashion

ID | possible_values
-- | --
GB020 | 1: present, 0: absent
GB021 | 1: present, 0: absent
GB022 | 1: present, 0: absent
GB023 | 1: present, 0: absent
GB024 | multistate 1: Num-N; 2: N-Num; 3: both.
GB025 | multistate 1: Dem-N; 2: N-Dem; 3: both.
GB026 | 1: present, 0: absent
GB027 | 1: present, 0: absent
GB028 | 1: present, 0: absent
GB030 | 1: present, 0: absent
GB031 | 1: present, 0: absent
GB035 | 1: present, 0: absent
GB036 | 1: present, 0: absent
GB037 | 1: present, 0: absent
GB038 | 1: present, 0: absent
GB039 | 1: present, 0: absent
GB041 | 1: present, 0: absent
GB042 | 1: present, 0: absent
GB043 | 1: present, 0: absent
GB044 | 1: present, 0: absent
GB046 | 1: present, 0: absent
GB047 | 1: present, 0: absent
GB048 | 1: present, 0: absent
GB049 | 1: present, 0: absent
GB051 | 1: present, 0: absent
GB052 | 1: present, 0: absent
GB053 | 1: present, 0: absent
GB054 | 1: present, 0: absent
GB057 | 1: present, 0: absent
GB058 | 1: present, 0: absent
GB059 | 1: present, 0: absent
GB065 | multistate 1:Possessor-Possessed; 2:Possessed-Possessor; 3: both
GB068 | 1: present, 0: absent
GB069 | 1: present, 0: absent
GB070 | 1: present, 0: absent
GB071 | 1: present, 0: absent
GB072 | 1: present, 0: absent
GB073 | 1: present, 0: absent
GB074 | 1: present, 0: absent
GB075 | 1: present, 0: absent
GB079 | 1: present, 0: absent
GB080 | 1: present, 0: absent
GB081 | 1: present, 0: absent
GB082 | 1: present, 0: absent
GB083 | 1: present, 0: absent
GB084 | 1: present, 0: absent
GB086 | 1: present, 0: absent
GB089 | 1: present, 0: absent
GB090 | 1: present, 0: absent
GB091 | 1: present, 0: absent
GB092 | 1: present, 0: absent
GB093 | 1: present, 0: absent
GB094 | 1: present, 0: absent
GB095 | 1: present, 0: absent
GB096 | 1: present, 0: absent
GB098 | 1: present, 0: absent
GB099 | 1: present, 0: absent
GB103 | 1: present, 0: absent
GB104 | 1: present, 0: absent
GB105 | 1: present, 0: absent
GB107 | 1: present, 0: absent
GB108 | 1: present, 0: absent
GB109 | 1: present, 0: absent
GB110 | 1: present, 0: absent
GB111 | 1: present, 0: absent
GB113 | 1: present, 0: absent
GB114 | 1: present, 0: absent
GB115 | 1: present, 0: absent
GB116 | 1: present, 0: absent
GB117 | 1: present, 0: absent
GB118 | 1: present, 0: absent
GB119 | 1: present, 0: absent
GB120 | 1: present, 0: absent
GB121 | 1: present, 0: absent
GB122 | 1: present, 0: absent
GB123 | 1: present, 0: absent
GB124 | 1: present, 0: absent
GB126 | 1: present, 0: absent
GB127 | 1: present, 0: absent
GB129 | 1: present, 0: absent
GB130 | multistate 1: SV; 2: VS; 3: both
GB131 | 1: present, 0: absent
GB132 | 1: present, 0: absent
GB133 | 1: present, 0: absent
GB134 | 1: present, 0: absent
GB135 | 1: present, 0: absent
GB136 | 1: present, 0: absent
GB137 | 1: present, 0: absent
GB138 | 1: present, 0: absent
GB139 | 1: present, 0: absent
GB140 | 1: present, 0: absent
GB146 | 1: present, 0: absent
GB147 | 1: present, 0: absent
GB148 | 1: present, 0: absent
GB149 | 1: present, 0: absent
GB150 | 1: present, 0: absent
GB151 | 1: present, 0: absent
GB152 | 1: present, 0: absent
GB155 | 1: present, 0: absent
GB156 | 1: present, 0: absent
GB158 | 1: present, 0: absent
GB159 | 1: present, 0: absent
GB160 | 1: present, 0: absent
GB165 | 1: present, 0: absent
GB166 | 1: present, 0: absent
GB167 | 1: present, 0: absent
GB170 | 1: present, 0: absent
GB171 | 1: present, 0: absent
GB172 | 1: present, 0: absent
GB177 | 1: present, 0: absent
GB184 | 1: present, 0: absent
GB185 | 1: present, 0: absent
GB186 | 1: present, 0: absent
GB187 | 1: present, 0: absent
GB188 | 1: present, 0: absent
GB192 | 1: present, 0: absent
GB193 | multistate 1: ANM - N; 2: N - ANM; 3: both; 0: they cannot be used attributively
GB196 | 1: present, 0: absent
GB197 | 1: present, 0: absent
GB198 | 1: present, 0: absent
GB203 | multistate 0: no UQ, 1: UQ-N, 2: N-UQ, 3: both
GB204 | 1: present, 0: absent
GB250 | 1: present, 0: absent
GB252 | 1: present, 0: absent
GB253 | 1: present, 0: absent
GB254 | 1: present, 0: absent
GB256 | 1: present, 0: absent
GB257 | 1: present, 0: absent
GB260 | 1: present, 0: absent
GB262 | 1: present, 0: absent
GB263 | 1: present, 0: absent
GB264 | 1: present, 0: absent
GB265 | 1: present, 0: absent
GB266 | 1: present, 0: absent
GB270 | 1: present, 0: absent
GB273 | 1: present, 0: absent
GB275 | 1: present, 0: absent
GB276 | 1: present, 0: absent
GB285 | 1: present, 0: absent
GB286 | 1: present, 0: absent
GB291 | 1: present, 0: absent
GB296 | 1: present, 0: absent
GB297 | 1: present, 0: absent
GB298 | 1: present, 0: absent
GB299 | 1: present, 0: absent
GB300 | 1: present, 0: absent
GB301 | 1: present, 0: absent
GB302 | 1: present, 0: absent
GB303 | 1: present, 0: absent
GB304 | 1: present, 0: absent
GB305 | 1: present, 0: absent
GB306 | 1: present, 0: absent
GB309 | 1: present, 0: absent
GB312 | 1: present, 0: absent
GB313 | 1: present, 0: absent
GB314 | 1: present, 0: absent
GB315 | 1: present, 0: absent
GB316 | 1: present, 0: absent
GB317 | 1: present, 0: absent
GB318 | 1: present, 0: absent
GB319 | 1: present, 0: absent
GB320 | 1: present, 0: absent
GB321 | 1: present, 0: absent
GB322 | 1: present, 0: absent
GB323 | 1: present, 0: absent
GB324 | 1: present, 0: absent
GB325 | 1: present, 0: absent
GB326 | 1: present, 0: absent
GB327 | 1: present, 0: absent
GB328 | 1: present, 0: absent
GB329 | 1: present, 0: absent
GB330 | 1: present, 0: absent
GB331 | 1: present, 0: absent
GB333 | 1: present, 0: absent
GB334 | 1: present, 0: absent
GB335 | 1: present, 0: absent
GB336 | 1: present, 0: absent
GB400 | 1: present, 0: absent
GB401 | 1: present, 0: absent
GB402 | 1: present, 0: absent
GB403 | 1: present, 0: absent
GB408 | 1: present, 0: absent
GB409 | 1: present, 0: absent
GB410 | 1: present, 0: absent
GB415 | 1: present, 0: absent
GB421 | 1: present, 0: absent
GB422 | 1: present, 0: absent
GB430 | 1: present, 0: absent
GB431 | 1: present, 0: absent
GB432 | 1: present, 0: absent
GB433 | 1: present, 0: absent
GB519 | 1: present, 0: absent
GB520 | 1: present, 0: absent
GB521 | 1: present, 0: absent
GB522 | 1: present, 0: absent

**Comment by HedvigS** [comment]
I.e. same information that is in "codes.csv" would be great if it was also in parameters.csv.

**Comment by xrotwang** [comment]
Why would you want to duplicate this information - and do that in an unspecified, somewhat hard to parse way? [`codes.csv`](https://github.com/cldf/cldf/tree/master/components/codes) is part of the CLDF spec, and grouping codes by feature ID isn't particularly difficult.

**Comment by HedvigS** [comment]
I thought it might be easier since that's how the coding sheets look. But sure, I can create that by group_by() in the scripts instead.

codes <- read_csv("../../cldf/codes.csv")  %>% 
  unite(possible_values_split, Name,  Description, sep = ": ") %>% 
  group_by(Parameter_ID) %>% 
  mutate(possible_values = paste0(possible_values_split, collapse = ", ")) %>% 
  distinct(Parameter_ID, possible_values)
